public/app/features/dashboard/utils/getPanelMenu.ts

```diff
@@ -179,6 +179,14 @@ export function getPanelMenu(
     onClick: (e: React.MouseEvent<any>) => onInspectPanel('json'),
   });
 
+  // Only show for editors
+  if (panel.plugin && dashboard.meta.canEdit && !panel.plugin.meta.skipDataQuery) {
```

Comment:
This if is quite obscure when we add `&& !panel.plugin.meta.skipDataQuery`. What does this mean?

Reply:
Let me know if I'm wrong here @ryantxu, but as I understand it, certain panel types don't need to make queries, for example the Text or News panels, so in those cases there's not really anything to debug and we don't want to show "Debug snapshot" as a panel menu option.

Reply:
@kaydelaney -- exactly.  In the current state the wizard only really helps debugging data+visualization issues, for panels that do not use data this would really be noise (as is)